src/Units/KaijuWarsKaiju.java

```diff
@@ -83,7 +83,7 @@ public class KaijuWarsKaiju
 
   private static final int KAIJU_COST = 0;
   private static final int MOVE_POWER = 2; // Dummy value
-  private static final double STAR_VALUE = 10.0;
+  private static final int STAR_VALUE = 50;
```

Comment:
Not that it matters much (because Kaiju) but why not 100? You did a x10 everywhere else.

Reply:
Kaiju can have a lot more than 10 HP, and I really didn't think about it much the first time.